### PR TITLE
Get lockmode by the storemode

### PR DESCRIPTION
**Pull request by zhangthen**
#1192

### CODE REVIEW

File: server/src/main/resources/file.conf [hunk 1]
```diff
@@ -82,11 +82,7 @@ store {
 }
 lock {
     ## the data row lock store mode: local_db、memory or db
-    mode = "memory"
-
-    memory{
-         ## store lock in memory of server
-    }
+    mode = ""
```
Comment (leizhiyuan) [inline server/src/main/resources/file.conf]: should we remove lock.mode from conf

Comment (zhangthen) [reply server/src/main/resources/file.conf]: > should we remove lock.mode from conf

I'll use “lock.mode”  in some other cases.

File: server/src/main/resources/file.conf [hunk 2]
```diff
@@ -82,11 +82,7 @@ store {
 }
 lock {
```
Comment (slievrly) [inline server/src/main/resources/file.conf]: I think  should delete the lock node, which is strongly associated with the store. As for the table name, we should use our default table name just like `undolog`, and do not need to allow user settings.

Comment (zhangthen) [reply server/src/main/resources/file.conf]: > I think should delete the lock node, which is strongly associated with the store. As for the table name, we should use our default table name just like `undolog`, and do not need to allow user settings.

I will store the row locks in the user's database, beside the undo_log table, and need a lock node to configure this switcher.

**Pull request merged**